index.js

```diff
@@ -0,0 +1,22 @@
+var Cli = require("matrix-appservice-bridge").Cli;
+var AppServiceRegistration = require("matrix-appservice-bridge").AppServiceRegistration;
+
+var MatrixGitterBridge = require("./lib/MatrixGitterBridge");
+
+new Cli({
+  registrationPath: "gitter-registration.yaml",
+  bridgeConfig: {
+    schema: "config/gitter-config-schema.yaml",
+  },
+  generateRegistration: function(reg, callback) {
+    reg.setHomeserverToken(AppServiceRegistration.generateToken());
```

Comment:
That was in a way intentional. I thought the point of the `id` key was to be meaningful to the end user, the HS administrator. So just filling in random junk there is unlikely to be helpful to them.